optimum/habana/transformers/generation/configuration_utils.py

```diff
@@ -27,10 +27,6 @@ class GaudiGenerationConfig(GenerationConfig):
         If negative (default=-1) pad to max if `static_shapes` is set. Else start with
         `shape = bucket_size * ceil(prompt_len/bucket_size)` and then grow space by `bucket_size` when needed.
         Only active if `static_shapes` is used. Can't be used with `reuse_cache`.
-    bucket_internal (`bool`, *optional*):
-        Split kv sequence into buckets in decode phase. It improves throughput when max_new_tokens is large.
```

Comment:
why do you remove the bucket_internal, can you add it back?

Reply:
Not sure, the cherry-pick ended without errors. I'll readd it